_includes/timeline_strip.html

```
@@ -58,22 +66,24 @@
 <figure class="tl-strip">
   <div class="tl-strip-row">
     {% assign _items = include.items | split: ";" %} {% for _it in _items %} {%
-    assign _p = _it | split: "|" %} {% assign _src = _p[0] | strip %} {% assign
-    _cap = _p[1] | strip %}
+    assign _p = _it | split: "|" %} {% if _p.size > 1 %} {% assign _src = _p[0]
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)